src/components/report/VariableControls.js

```diff
@@ -64,11 +65,13 @@ export default {
 			let counter = 0;
 			let maxProps = 6;
 			for (const axis of Object.values(this.axes)) {
-				styles += `${glue} "${axis.id}" ${axis.current}`;
-				glue = ",";
-				// Poor man's code formatting
-				if (++counter % maxProps === 0) {
-					glue = `,\n                        `;
+				if (this.currentStates[axis.id] !== false) {
```

Comment:
Could this be extracted into a function, so it can be unit tested? There's more and more logic creeping in here, so I think it's worth it to start testing this. WDYT?

Reply:
Sure, we might be hitting that point now. Could you help me set this up? Then I can perhaps move more logic to testable functions.